lib/src/dispatch_settings.dart

```diff
@@ -0,0 +1,128 @@
+bool _whereNotOlderThanADay(Map<String, String> action) =>
+    DispatchSettings.whereNotOlderThan(
+      const Duration(
+        hours: 23,
+        minutes: 59,
+        seconds: 59,
+      ),
+    )(action);
+
+bool _notOlderThan(
+  Map<String, String> action,
+  DateTime now,
+  Duration duration,
+) {
+  final date = action['cdt']!;
```

Comment:
Following the discussion in https://github.com/Floating-Dartists/matomo-tracker/pull/97#discussion_r1211512391
I'd suggest something like this to get rid of the `!`:
```dart
if (action['cdt'] case final date?) {
  return now.difference(DateTime.parse(date)) <= duration;
}
return false;
```

Reply:
It's all new from Dart 3 😉